Automattic/wp-calypso
# Login: Implement login by site url

Part of pdDR7T-12Q-p2

Related to #83359 

## Proposed Changes

This allows logging in by site address.

## Testing Instructions

1. Apply D130157-code.
2. Try to log in using the live link below and use a site that is owned by a single person. Ping me if you want one.
3. Type the password of the owner of that account, you should be logged in.
